api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -193,6 +193,10 @@ message HttpConnectionManager {
     //   Such a constraint is inherent to OpenCensus itself. It cannot be overcome without changes
     //   on OpenCensus side.
     config.trace.v3.Tracing.Http provider = 9;
+
+    // Trace reason is decided by request_id by default. If true, these sampling strategy is delegated
+    // to the configured tracing provider.
+    bool delegate_sampling = 10;
```

Comment:
Not sure if `delegate_sampling` is the best name here.
IIUC when the flag is enabled, all requests will be traced (i.e., handed off to the provider).

It would be good to note that if the flag is enabled, the `x-request-id` header will not be modified (even if provided).

Reply:
changed to `bypass_sampling_with_request_id`